carbonmark/components/pages/Portfolio/ConnectedWallet/styles.ts

```diff
@@ -0,0 +1,18 @@
+import { css } from "@emotion/css";
+
+export const cardbody = css`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+export const marginLeft = css`
+  margin-left: 5px;
+`;
```

Comment:
Some small things here:
* In the designs the spacing is `8px`
* We usually use `rem` as a unit. `1rem` is the same as `10px` and we can therefore use `0.8rem` here.
* We can use a flexbox gap instead of left margins

```suggestion
export const cardbody = css`
  display: flex;
  gap: 0.8rem;
  flex-direction: row;
  justify-content: space-between;
  align-items: center;
`;
```